kani-compiler/src/codegen_cprover_gotoc/codegen/typ.rs

```diff
@@ -354,9 +355,9 @@ impl<'tcx> GotocCtx<'tcx> {
         Type::unit().to_typedef(inner_name)
     }
 
-    /// This will codegen the raw pointer to the trait data.
+    /// This will codegen the raw pointer to the data portion of a vtable fat pointer.
```

Comment:
This comment confused me

Reply:
Do you prefer the old comment or do you have any suggestions?